docs/user-guide/localization.md

```diff
@@ -0,0 +1,74 @@
+### Localization in ADF
+
+Localization is the process of making something local in character or restricting it to a particular place. 
+
+Dates are not written the same around the world. That is where localizing a date comes in handy. ADF lets you dynamically change the way dates are written in your app so that they can adapt to to a specific region.
+
+## Contents
+
+-   [Setting up the configuration in your app](#setting-up-the-configuration-in-your-app)
+-   [Adding language support](#adding-language-support)
+-   [Using the localized date pipe](#using-the-localized-date-pipe)
+-   [Using the time ago pipe](#using-the-time-ago-pipe)
+
+## Setting up the configuration in your app
+
+You can overwrite the default values of this pipe by adding these properties to your ```app.config.json```:
+
+```json
+ "dateValues": {
+   "defaultDateFormat": "mediumDate",
+   "defaultDateTimeFormat": "MMM d, y, h:mm",
```

Comment:
maybe needs `H` here too?
```suggestion
   "defaultDateTimeFormat": "MMM d, y, H:mm",
```

Reply:
I guess the bug that we are fixing is ok